### PR TITLE
pinned dependencies

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: constraints.txt [hunk 1]
```diff
@@ -0,0 +1,82 @@
+aniso8601==3.0.2
```
Comment (andrevmatos) [inline constraints.txt]: May we call this file `requirements-lock.txt`, in order to comply with other dependency management systems like `npm`, and also keeping it lexically close to the other `requirements` files

Comment (hackaugusto) [reply constraints.txt]: I don't have strong opinion about this, @ulope is there a standard for this?

**Comment by hackaugusto** [comment]
this should fix https://github.com/raiden-network/raiden/issues/2460

File: constraints.txt (hunk 1 above)
Comment (ulope) [reply constraints.txt]: I don't have a strong preference either (and don't know of a standard). Bu am leaning slightly towards keeping `constraints.txt` as that's the name used by pip itself:

```
-c, --constraint <file>     Constrain versions using the given constraints file. This option can be used multiple times.
```

**Comment by ulope** [comment]
> this should fix #2460

I'm not sure it will. But let's see :)

**Comment by hackaugusto** [comment]
@ulope did this fix the problem? or do you still think this was not a solution?

@palango complained that because we do use the `constraints.txt` keeping it up in the services repo has been a burden.
@LefterisJP would also like to have pinned version in the `requirements.txt` instead of the `constraints.txt`

So, I added this to fix dependencies problems because I wanted something closer to deterministic builds. But if it's has being a burden instead of a solution we can revert it.

**Comment by hackaugusto** [comment]
btw, @LefterisJP an alternative is to use a pipfile.lock https://github.com/pypa/pipfile

**Pull request merged**